.ddev/.setup/scripts/utils.sh

```
@@ -89,32 +138,48 @@ function check_typo3_version() {
+  install_composer_packages
 }
 
 # Function to perform post-setup tasks for TYPO3 installation.
 # It changes the current directory to the base path, sets the TYPO3 installation database name,
 # and calls the appropriate post-setup function based on the TYPO3 version.
 # After that, it imports data and updates TYPO3.
 function post_setup() {
-  message blue "Post Setup for TYPO3 $VERSION"
+  prepare_acceptance_testing
+
   cd $BASE_PATH
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)